www/faq.php

```diff
@@ -75,38 +75,52 @@
 
 	<tr><td class="textcontent">
 	<p>
-	You should be familiar with the <a href="https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/ports.html">Ports</a>
-	section of <a href="https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/index.html">The FreeBSD Handbook</a>.
-	Pay careful attention to the difference between a port and a package.
+	Pay careful attention to the differences between a port and a package.
+	<p>
+	Blogged by the FreeBSD Foundation in August 2024:
+	<a href="https://freebsdfoundation.org/blog/freebsd-ports-and-packages-what-you-need-to-know/">
+	FreeBSD Ports and Packages: What you need to know</a>. More generally,
+	please be familiar with the <a href="https://docs.freebsd.org/en/books/handbook/ports/">
+	Ports</a> section of the <a href="https://docs.freebsd.org/en/books/handbook/">
+	FreeBSD Handbook</a>.
 
 	<h2>Definitions for the hopelessly lazy</h2>
 
 	<h3 id="port">What is a port?</h3>
 	<p>
-	If you install the port, the source will be downloaded, patched if necessary,
-	compiled and installed.  You will have a chance to set any optional configuration
-	items particular to that port. If the port is dependent upon another port,
-	that port will also be installed.  After installing, ports are identical to
-	packages.
-
+	If you install a port, the source will be downloaded, patched if 
+	necessary, compiled and installed. If the port has any default
```

Comment:
"compiled, and installed"

Oxford commas, all, the, way.

Reply:
```suggestion
	necessary, compiled, and installed. If the port has any default
```